src/SubtitleOctopus.cpp

```diff
@@ -247,6 +247,8 @@ class SubtitleOctopus {
 
     int status;
 
+    char defaultFont[256];
```

Comment:
Any particular reason for the fixed-sized buffer? Can we not use `std::string` here, or `std::unique_ptr`, or at worst, just `strdup` and an explicit `free` in the destructor?

Reply:
No particular reason, this is how it was done originally for simplicity.
I'll change the code after rearranging commits from #145

Reply:
No particular reason, this is how it was done originally for simplicity.
Will change this after rearranging #145 commits.

Reply:
This is now using std::string